ajax/libs/bootstrap-rating-input/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "bootstrap-rating-input",
+  "filename": "bootstrap-rating-input.min.js",
+  "version": "0.4.0",
+  "license": "MIT",
+  "author": "Javier Toledo <javier@theagilemonkeys.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/javiertoledo/bootstrap-rating-input"
+  },
+  "description": "A Lightweight Bootstrap Rating Input",
+  "keywords": [
+    "bootstrap",
+    "rating",
+    "input"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/javiertoledo/bootstrap-rating-input.git",
+    "fileMap": [
+      {
+        "basePath": "bulid",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "**/*"
```

Comment:
@extend1994 It's a little bit not safe or no sense to use `**/*` under a non-dist folder.